src/main/webapp/js/instructorCourseEdit.es6

```diff
@@ -512,6 +511,22 @@ $(document).ready(() => {
     const editLinks = $('a[id^=instrEditLink]');
     instructorSize = editLinks.length;
     $(editLinks).click(editFormRequest);
+
+    const clickHandlerMap = new Map();
+    clickHandlerMap.set('.hide-tune-section-permissions', hideTuneSectionPermissionsDiv);
+    clickHandlerMap.set('.show-tune-section-permissions', showTuneSectionPermissionsDiv);
+    clickHandlerMap.set('.hide-tune-session-permissions', hideTuneSessionnPermissionsDiv);
+    clickHandlerMap.set('.show-tune-session-permissions', showTuneSessionnPermissionsDiv);
+
+    /* eslint-disable no-restricted-syntax */
+    for (const [className, clickHandler] of clickHandlerMap) {
+        $(document).on('click', className, (e) => {
+            const instructorIndex = parseInt($(e.target).closest('a').data('instructorindex'), 10);
+            const panelIndex = parseInt($(e.target).closest('a').data('panelindex'), 10);
```

Comment:
I know it was my code, but you can make it simpler like:
```js
const *Index = $(e.target).closest('a').data('*index');
```
Better yet, see if this works:
```js
const *Index = $(e.currentTarget).data('*index');
```